### PR TITLE
Skip Lighthouse audit of off-site page

### PR DESCRIPTION
**Pull request by palewire**
Screenshot from 2022-01-09 14-23-50

### CODE REVIEW

**Comment by palewire** [comment]
My edit to the config file [doesn't seem](https://github.com/jupyter/jupyter.github.io/runs/4755372736?check_suite_focus=true) to have worked. What am I doing wrong here?

Screenshot from 2022-01-09 14-29-07

File: .github/workflows/lighthouserc.json [hunk 1]
```diff
@@ -3,7 +3,8 @@
     "collect": {
       "staticDistDir": "./_site",
       "autodiscoverUrlBlocklist": [
-        "conduct.html"
+        "conduct.html",
+        "https://docs.jupyter.org/en/latest/"
```
Comment (krassowski) [inline .github/workflows/lighthouserc.json:7]: I think it is entering it from some redirect, maybe from try `"documentation.html"` or "documentation"` instead?

**Comment by palewire** [comment]
That did it. This is ready for your review, @krassowski.

**Review by krassowski — APPROVED** [review]
Thanks.

**Pull request merged**